test/functional/sqlsrv/MsCommon.inc

```diff
@@ -32,6 +33,18 @@ function setUTF8Data($val)
     $usingUTF8data = $val;
 }
 
+function testAnsiData()
+{
+    global $testingAnsiData;
+    return $testingAnsiData;
+}
+
+function setTestAnsiData($val)
+{
+    global $testingAnsiData;
+    $testingAnsiData = $val;
+}
+
 function testMode()
```

Comment:
Is it possible to refactor the above four methods? Use constants (like enums) or boolean variables to distinguish the two test cases (either utf-8 or not)?

Reply:
changes have been reverted